### DISCUSSION TITLE
Is it possible to use Authentication of Supabase and Database and for data storing serverpod?

### DISCUSSION THREAD

**Discussion by iampopal** [serverpod/serverpod#2812] [Q&A]
Because we already have our user account and profile saved in supabase and we might using it in future too. is it possible to use Authentication of Supabase with Just Database for Data And Files Storing Serverpod.

**Comment by Isakdl** [comment]
There are no premade integrations for this, but it is definitely possible if you build your own integration, see the documentations on how to do that here: https://docs.serverpod.dev/concepts/authentication/providers/custom-providers

**Reply by iampopal** [reply]
We an app to launch quickly. If possible from community someone help me prepare a basic simple of supabase and serveepod authentication integration will be a big help so far.
